Dockerfile.windows

```diff
@@ -55,6 +55,15 @@ RUN Write-Host ('Installing Visual C++ Redistributable Package'); `
     Copy-Item -Path /Windows/System32/vccorlib140.dll -Destination /work/out/bin/; `
     Copy-Item -Path /Windows/System32/vcruntime140.dll -Destination /work/out/bin/;
 
+#
+# Install OpenSSL
+#
+ADD https://slproweb.com/download/Win64OpenSSL-1_1_1L.msi /local/Win64OpenSSL-1_1_1L.msi
```

Comment:
Both mbedtls and OpenSSL are rarely used on Windows and this will be surprising to users either way; the Windows SSL library is called [Secure Channel](https://docs.microsoft.com/en-us/windows/win32/secauthn/secure-channel). Is there any way to use Secure Channel with fluentbit?

(For example, I don't think either off them will pick up enterprise certificates that the user has installed.)

Reply:
> Is there any way to use Secure Channel with fluentbit?

Fluent Bit checks for OpenSSL specifically: https://github.com/fluent/fluent-bit/blob/1d529d9fbc5de5559943ad5fd4e4c070fa319be7/CMakeLists.txt#L468-L473

Reply:
> If there's no preference, would using the [libraries from the `curl` project](https://github.com/curl/curl-for-win) be safer?

I'm not sure I understand the curl project -- what makes that safer? It looks like it'd also install other things we don't need. And I'm not sure that cmake's `find_package` (what [Fluent Bit uses](https://github.com/fluent/fluent-bit/blob/1d529d9fbc5de5559943ad5fd4e4c070fa319be7/CMakeLists.txt#L468-L473)) would work correctly if OpenSSL is a linked lib of something else

Reply:
Added hash check!

Reply:
The `curl` project is better known than Shining Light, I guess. But it looks like it's more of a hassle, and now that we check the hash, it should be safe.